### PR TITLE
[hbase098] Fix checkstyle for HBase 0.98.x binding

### PR DESCRIPTION
**Pull request by risdenk**
Relates to #267 and supersedes #484

### CODE REVIEW

**Comment by risdenk** [comment]
I'll rebase this in a few minutes and push the changes. This is the only remaining binding that doesn't enforce checkstyle!

**Comment by risdenk** [comment]
@busbey if you have any spare cycles to review this PR that would be greatly appreciated. It is the last of the checkstyle changes.

**Comment by risdenk** [comment]
I've been using https://github.com/brianfrankcooper/YCSB/pull/902/files?w=1 to check it to ignore the whitespace only changes.

File: hbase098/src/main/java/com/yahoo/ycsb/db/HBaseClient.java [hunk 1]
```diff
@@ -22,576 +22,460 @@
...
 
-/* For customized vim control
- * set autoindent
- * set si
- * set shiftwidth=4
-*/
+    for (Thread t : allthreads) {
+      try {
+        t.join();
+      } catch (InterruptedException ignored) {
```
Comment (busbey) [inline hbase098/src/main/java/com/yahoo/ycsb/db/HBaseClient.java]: This is incorrect behavior. At a minimum the interrupted status for the current thread should be set.

It's a preexisting problem, not something from this fix, so I'd be fine with leaving it so long as there's an issue filed to fix it.

**Comment by risdenk** [comment]
Thanks @busbey!

**Pull request merged**